sdk/communication/tests.yml

```diff
@@ -25,11 +25,12 @@ extends:
       ServiceDirectory: communication
```

Comment:
Should this test pipeline even exist given it is split out by package already?

Reply:
Seems like a no to me